cmake/testing.cmake

```diff
@@ -142,6 +142,10 @@ f3d_test(NAME TestVolumeComp DATA vase_4comp.vti ARGS -vb --comp=3 LONG_TIMEOUT)
 f3d_test(NAME TestVolumeDirect DATA vase_4comp.vti ARGS -vb --comp=-2 LONG_TIMEOUT)
 f3d_test(NAME TestVolumeCells DATA waveletArrays.vti ARGS -vb --cells LONG_TIMEOUT)
 f3d_test(NAME TestVolumeNonScalars DATA waveletArrays.vti ARGS -vb --scalars=RandomPointScalars LONG_TIMEOUT)
+f3d_test(NAME TestTextureNormal DATA WaterBottle.glb ARGS --geometry-only --texture-normal=${CMAKE_SOURCE_DIR}/testing/data/normal.png --normal-scale=0.1 DEFAULT_LIGHTS)
+f3d_test(NAME TestTextureMaterial DATA WaterBottle.glb ARGS --geometry-only --texture-material=${CMAKE_SOURCE_DIR}/testing/data/red.jpg --roughness=1 --metallic=1 DEFAULT_LIGHTS) # roughness and metallic have no effect ?
```

Comment:
@Meakk PTAL

Reply:
You are setting the material texture with a texture having only a red pixel. So you have occlusion=red=1, roughness=green=0, metallic=blue=0.  
The values are multiplied by the constant value given by the user, but since the value of roughness and metallic is 0, you can give whateever you want in the constant values, the result is the same.

Reply:
Indeed, this was the cause of my confusion.

However, are you sure that:
>The values are multiplied by the constant value given by the user, 
?

Here is another test with the following texture:

```
R=1
G=0.5
B=0.5
```
![red_mod](https://user-images.githubusercontent.com/3129530/180605134-034f029b-f66d-46e3-8c69-3484a05d7878.jpg)

`f3d  --geometry-only --texture-material=red_mod.jpg`:
![a](https://user-images.githubusercontent.com/3129530/180605171-a62a03d0-f454-46d1-ab18-d6c99157f107.png)

`f3d  --geometry-only --texture-material=red_mod.jpg --roughness=1`:
![a](https://user-images.githubusercontent.com/3129530/180605196-2d1bb1da-c198-4fe3-86ff-cb093cc7a4c4.png)


If that is indeed a multiplication, shouldn't these images be the same ?

Reply:
In the first case, you have:
- default roughness (0.3) * G channel (0.5) = 0.15
- custom roughness (1) * G channel (0.5) = 0.5

In both cases, the metallic value is 0 (because the default value is 0 so the texture has no effect)

Reply:
Ok. I understand now.

Do we want this behavior ? Someone using --texture-material would not expect the default roughness and default metallic to multiply its own texture material ?

We could either:
- Keep this behavior but with better documentation
- Change the default to zero but set it in the config file
- Change the default to -1 and implement a smart default

Let me know what you think

Reply:
IMHO, what we should do is change the default values to roughess=1, metallic=1, when the material texture is set, so the multiplier does nothing, but the user can still modify it if he wants to.

Reply:
This is the third solution.

Reply:
1, will be implemented, but not in this PR, see #307